modules/creating-serverless-apps-kn.adoc

```diff
@@ -24,21 +24,22 @@ $ kn service create <service-name> --image <image> --env <key=value>
 [source,terminal]
 [source,terminal]
 ----
-$ kn service create event-display \
-    --image quay.io/openshift-knative/knative-eventing-sources-event-display:latest
+$ kn service create hello \
+    --env RESPONSE='Hello Serverless!' \
+    --image docker.io/openshift/hello-openshift
 ----
 +
 .Example output
 [source,terminal]
 ----
-Creating service 'event-display' in namespace 'default':
+Creating service 'hello' in namespace 'default':
 
   0.271s The Route is still working to reflect the latest desired specification.
-  0.580s Configuration "event-display" is waiting for a Revision to become ready.
+  0.580s Configuration "hello" is waiting for a Revision to become ready.
   3.857s ...
   3.861s Ingress has not yet been reconciled.
   4.270s Ready to serve.
 
-Service 'event-display' created with latest revision 'event-display-bxshg-1' and URL:
-http://event-display-default.apps-crc.testing
+Service 'hello' created with latest revision 'hello-bxshg-1' and URL:
+http://hello-default.apps-crc.testing
 ----
```

Comment:
Can we keep the name of the example service here as `event-display` please; I think this is the service name that is generally used for examples in our docs and it seems confusing to change it now for no reason? e.g. https://docs.openshift.com/container-platform/4.7/serverless/event_sources/serverless-pingsource.html